react/features/rtcstats/RTCStats.ts

```diff
@@ -96,7 +124,23 @@ class RTCStats {
      * @returns {boolean}
      */
     isInitialized() {
-        return this.initialized;
+        return this.options !== undefined;
+    }
+
+    /**
+     * Resets the rtcstats.
+     *
+     * @returns {void}
+     */
+    reset() {
+        delete this.options;
+
+        // FIXME unwrap the peer connection.
```

Comment:
As indicated above I think I would rather try to have the PC wrapper hold references into what it is suppose to call, than to try to unwrap everything here.

Reply:
Lets remove the comment, if this is no longer needed.